modules/apps/site-initializer/site-initializer-raylife/src/main/zippableResources/fragments/raylife/fragments/list-item-with-icon/index.html

```diff
@@ -0,0 +1,17 @@
+<li class="fragment_71 d-flex justify-content-center flex-column align-items-center">
```

Comment:
```suggestion
<li class="align-items-center d-flex flex-column insurance-faq  justify-content-center">
```

fragment_71 should be renamed, I suggest to put `insurance-faq` 

Insurance faq: "Insurance Frequently asked question"

Also, the folder could be renamed from `list-item-with-icon` to `insurance-faq`

Why: Whenever possible, we need to bring clarify to the code and assets. It's easy to read and understand what's means: list item with icon, it could be anything. Different from read Insurance FAQ

Reply:
Thank you for this suggestion. I couldn't think about a better name to it.